dependencies/versions_test.go

```diff
@@ -55,14 +63,20 @@ func TestSanity(t *testing.T) {
 func TestSemverVersions(t *testing.T) {
 	a := Version{"1.0.0", Semver}
 	b := Version{"2.0.0", Semver}
+
+	// nolint: errcheck
```

Comment:
I think you may be able to disable errcheck on all test files in the linting config